src/resources/certifyStatementResource.ts

```diff
@@ -0,0 +1,37 @@
+export const CertifyStatementResource = [{
+  entityType: 'BC',
+  displayName: 'Benefit Company',
+  certifyStatementHeader: 'the Completing Party, have examined the Articles and ' +
+  'Incorporation Agreement applicable to the company that is to be incorporated ' +
+  'by the filing of this Incorporation Application and confirm that:',
+  certifyStatements: [
+    'the Articles and the Incorporation Agreement both contain a signature line for ' +
+    'each person identified as an incorporator in the Incorporation Application with ' +
+    'the name of that person set out legibly under the signature line,',
+    'an original signature has been placed on each of those signature lines, and',
+    'I have no reason to believe that the signature placed on a signature line is not ' +
+    'the signature of the person whose name is set out under that signature line.'
+  ],
+  certifyClause: 'It is an offence to make a false or misleading statement in respect ' +
+  'of a material fact in a record submitted to the Corporate Registry for filing. ' +
+  'See section 427 of the Business Corporations Act.'
+},
+{
+  entityType: 'CP',
+  displayName: 'Cooperatives',
+  certifyStatementHeader: 'the Completing Party, have examined the Articles and ' +
+  'Incorporation Agreement applicable to the company that is to be incorporated ' +
+  'by the filing of this Incorporation Application and confirm that:',
+  certifyStatements: [
+    'the Articles and the Incorporation Agreement both contain a signature line for ' +
+    'each person identified as an incorporator in the Incorporation Application with ' +
+    'the name of that person set out legibly under the signature line,',
+    'an original signature has been placed on each of those signature lines, and',
+    'I have no reason to believe that the signature placed on a signature line is not ' +
+    'the signature of the person whose name is set out under that signature line.'
+  ],
+  certifyClause: 'It is an offence to make a false or misleading statement in respect ' +
+  'of a material fact in a record submitted to the Corporate Registry for filing. ' +
+  'See section 427 of the Business Corporations Act.'
```

Comment:
This is not the correct text/reference for Cooperatives. We need to remember that this will need to be updated when we implement Coops.

Reply:
This is a placeholder for coops. Infact the text for bcomp is also not right from what I heard yesterday from Linda. The text will change for both coops and bcomp once the BA team has finalized the requirements. This is what we have at present in the UX pin